### PR TITLE
Persistant file data

### PR DESCRIPTION
**Pull request by george827**
Hello, In this feature branch we,

### **Highlights**

- [ ] On exit from the app save all the data (books, people, and rentals) into JSON files
- [ ] On startup of the app load all data from files (`rentals.json, books.json, people.json` ).

### CODE REVIEW

**Comment by TedLivist** [comment]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @george827 and @tsohleDev,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Some of the links (Usage) in your Table of Contents on the README are broken. Kindly add corresponding sections for them and link them properly. Remove the rest (like Run Tests and Deployment) if you have no tests or deployment now.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app.rb [hunk 1]
```diff
@@ -2,11 +2,21 @@
...
 class App
+  attr_reader :books, :people, :rentals
+
   def initialize
-    @books = []
-    @people = []
+    # read from json
+    @books = Writter.morph(JSON.parse(File.read('books.json')), 'books.json')
+    @people = Writter.morph(JSON.parse(File.read('people.json')), 'people.json')
+    @rentals = Writter.morph(JSON.parse(File.read('rentals.json')), 'rentals.json')
```
Comment (julie-ify) [inline app.rb]: - [ ] When I remove all the storage files and run your app I get a no file error. Kindly ensure that scenarios, where the JSON files do not exist, are handled properly - you can create the files before continuing with the options list, or at the moment when the data should be stored you should create the file but there should be no errors when files do not exist.

  image

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @george827 and @tsohleDev,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by muneebulrehman — APPROVED** [review]
Hi @george827 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**